netlify/functions/handler.js

```diff
@@ -0,0 +1,45 @@
+const { graphql } = require('@octokit/graphql')
+
+exports.handler = async function (event, context) {
+  if (event.httpMethod == 'POST') {
+    const { title, feedback } = JSON.parse(event.body);
+    const repositoryID = "MDEwOlJlcG9zaXRvcnkzNDc2MjE1NTk=";
+    const categoryID = "DIC_kwDOFLhIt84B_T4d"
+    try {
+      const createDiscussion = await graphql(`
+        mutation { 
+            createDiscussion(input:{repositoryId:"${repositoryID}", categoryId:"${categoryID}", title:"${title}", body:"${feedback}"}){
+             discussion{
+               url
+             }
+           }
+        }`,
+        {
+          owner: 'asyncapi',
+          repo: 'community',
+          headers: {
+            authorization: `token ${process.env.GITHUB_TOKEN_CREATE_DISCUSSION}`,
+          },
+        })
+      let url = createDiscussion.createDiscussion.discussion.url;
+      // res.status(200).json({ url: url, result: "Feedback posted in Discussion" });
+      return {
+        statusCode: 200,
+        body: JSON.stringify({
+          url: url,
```

Comment:
That's the purpose why I extracted the link from the GraphQL API, but I'm not getting an idea on how to show up the link to the user so that it perfectly fits in the design. Do you have any idea on how to show the URL to the user?

Reply:
In the information where you are thanked for the feedback there should be this url. And the thanks themselves should be visible all the time after sending the feedback, without any timeout.

Reply:
If I removed timeout from the function, it will remain always on `thanks` box and will not change for further pages for that browser window. This is because while rendering different doc pages, we are using `Link` component of Nextjs and it provides client-side navigation and static linking of one page to another. Hence, the Feedback Component treats them as the same page. So a timeout is made so that a user can submit multiple feedback for different pages.  

Regarding showing the link, I will try to add in between text of *thanking for feedback*.

Reply:
> If I removed timeout from the function, it will remain always on thanks box and will not change for further pages for that browser window. This is because while rendering different doc pages, we are using Link component of Nextjs and it provides client-side navigation and static linking of one page to another. Hence, the Feedback Component treats them as the same page. So a timeout is made so that a user can submit multiple feedback for different pages.

So, check by NextJS router that page changed and reset state for the feedback :)

Reply:
> So, check by NextJS router that page changed and reset state for the feedback :)

Thanks, it worked. I tested on my local and it is working fine. I will push the required changes in the next commit.

Reply:
@magicmatatjahu @akshatnema there was no intention to show the user link to anonymous feedback. Can you share a screen shot how did you implement it?